gradle.properties

```diff
@@ -1,3 +1,3 @@
 package=co.com.bancolombia
-systemProp.version=2.4.9
+systemProp.version=2.4.10-SNAPSHOT
```

Comment:
Please rollback this change

Reply:
Hi, ready the rollback, thanks for the feedback